### PR TITLE
[parser] GCP parser to gather license codes from instances

### PR DESCRIPTION
**Pull request by SteveHNH**
This is needed to identify marketplace systems in the GCP cloud.

Signed-off-by: Stephen Adams <tsadams@gmail.com>

### CODE REVIEW

File: insights/parsers/gcp_license_codes.py
Comment (bfahr) [inline insights/parsers/gcp_license_codes.py:3]: @SteveHNH please include the `.rst` file in the directory `insights/docs/share_parsers_catalog/` so that this new parser will be included in the documentation build.

File: insights/parsers/gcp_license_codes.py
Comment (bfahr) [inline insights/parsers/gcp_license_codes.py:32]: Since you are making changes could you add ParseException to this section of the doc since you raises it as well? Thx.

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/parsers/gcp_license_codes.py
Comment (SteveHNH) [reply insights/parsers/gcp_license_codes.py:3]: Ah! i keep forgetting that part. on it

File: insights/parsers/gcp_license_codes.py
Comment (SteveHNH) [reply insights/parsers/gcp_license_codes.py:32]: oh snap. yep

**Review by bfahr — APPROVED** [review]
Approved pending approval of the spec.

**Pull request merged**
